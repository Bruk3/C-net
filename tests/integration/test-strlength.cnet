
void foo(string[] a, int i){
    stdout.writeln(a[i]);
    stdout.writeln(soi(a[i].slength()));
    return;
}

int main (){
    string[] arr = new string[3] {"cnet","is", "cool"};
    int i =0;
    while (i < arr.alength()){
        foo(arr,i);
        i = i+1;
    }
    return 0;
}