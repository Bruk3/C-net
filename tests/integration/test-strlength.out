cnet
4
is
2
cool
4